model_demos/README.md

```diff
@@ -50,6 +50,7 @@ python cv_demos/resnet/pytorch_resnet.py
 |   [GPT Neo](nlp_demos/gptneo/)                               |     GS, WH   | v0.10.5 |
 |   [Hand Landmark](cv_demos/landmark/)                        |         WH   | v0.10.5 |
 |   [HRNet](cv_demos/hrnet/)                                   |     GS, WH   | v0.10.5 |
+|   [HarDNet](cv_demos/hardnet/)                               |     GS, WH   | v0.11.0-alpha |
```

Comment:
Sorry being nit-picky :) maybe move this one up one spot to be before HRNet for alphabetical order

Reply:
done https://github.com/tenstorrent/tt-buda-demos/pull/52/commits/8dbde32d18f764a0ad98cb90019e7b58911f7245